src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -227,17 +248,38 @@ public int read(ByteBuffer dst) throws IOException {
             src = fetch(position);
             msWaitingForData += waitingForData.elapsed(TimeUnit.MILLISECONDS);
         } catch (InterruptedException e) {
+            // Restore interrupted status
+            Thread.currentThread().interrupt();
```

Comment:
Could you explain this?  I'm not very familiar with how interruptions work.  Wouldn't being interrupted have set the interrupt status?  Why do we want to be set as interrupted if we are going to ignore the interrupt and return 0?